integration/mock-client/blockchain/bsn-irita.go

```diff
@@ -0,0 +1,33 @@
+package blockchain
+
+import (
+	"fmt"
+)
+
+func handleBSNIritaRequest(msg JsonrpcMessage) ([]JsonrpcMessage, error) {
+	switch msg.Method {
+	case "query_status":
+		return handleQueryStatus(msg)
+
+	case "query_blockResults":
+		return handleQueryBlockResults(msg)
+
+	case "query_serviceRequest":
+		return handleQueryServiceRequest(msg)
+
+	default:
+		return nil, fmt.Errorf("unexpected method: %v", msg.Method)
+	}
+}
+
+func handleQueryStatus(msg JsonrpcMessage) ([]JsonrpcMessage, error) {
+	return GetCannedResponses("birita", msg)
+}
+
+func handleQueryBlockResults(msg JsonrpcMessage) ([]JsonrpcMessage, error) {
+	return GetCannedResponses("birita", msg)
+}
+
+func handleQueryServiceRequest(msg JsonrpcMessage) ([]JsonrpcMessage, error) {
+	return GetCannedResponses("birita", msg)
+}
```

Comment:
I've been unable to test due to dependency issues, but this should automatically fetch the canned responses without this file. Have you tried this?

Reply:
This file only wraps some handlers for unit tests.

Reply:
Thanks for your review. In addition, I maked some optimization and added the integration test.

Reply:
Since they will just be static files, I don't think it will be necessary for unit tests for these. So ultimately I think it's best to get rid of this file, if possible.

Thanks for making the additions! It still seems like this PR needs to be rebased with master though.

Reply:
OK, Thanks. Pushed. 

As for this file you mentioned, there is a method named handleBSNIritaRequest, which is invoked by HandleRequest in common.go and remained. What do you think please?

Reply:
We should be able to remove the reference of `handleBSNIritaRequest()` in common.go. It should automatically read the canned responses then.

Reply:
Yes, removing is possible. But I was considering consistency with other chains and extensibility for future.

Reply:
Mock client updated for integration tests. 

Your review expected. Thanks.